### ISSUE TITLE
Entry in robots.txt that shouldn't be there

### ISSUE THREAD

**Issue by ade-akinyede** [alaca/WP-Performance#21]
Running a site audit, I get an error saying "Robots.txt file has format errors" caused by this entry:
```
<!-- WP Performance error: both head and body tag should be present in template file -->
```
While this is likely an issue, it shouldn't be in the robots.txt.

**Comment by alaca** [comment]
Hi, yes this is a WPP issue. Do you use any plugin to generate robots.txt file?

**Comment by ade-akinyede** [comment]
Not explicitly. I was thinking of doing this recently with clearfy plugin, but not sure if it'll create a clash with WPP?

**Comment by alaca** [comment]
Yes for sure. Everything generated dynamically could clash with WPP.  
I'll check it out as soon as I catch some time. 

Thank you for letting me know.

**Comment by ade-akinyede** [comment]
Ok, great :)
